src/components/contactForm/ContactForm.tsx

```diff
@@ -17,10 +18,11 @@ const ContactForm = () => {
 
   const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
-
+    setDisabled(true);
```

Comment:
i would also consider adding a loader inside the button , not just disabled state